app.py

```diff
@@ -78,7 +79,7 @@ def subs2json(subs):
 def find_request_errors(start_time, end_time, text, crop, resolution, episode):
     if end_time <= start_time:
         return 'end time must be after start time'
-    if end_time - start_time > 10:
+    if end_time - start_time > 10000:
```

Comment:
I switched from seconds to milliseconds to stay consistent with subtitles library